form.html

```diff
@@ -5,23 +5,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link type="Stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
     <title>Mystery Melody</title>
 </head>
 
-<body>
+<body class="backgroundImage">
+
     <section id="container">
         <section id="form">
+            <span class="mainText">
 
             <section>
                 <h1>What would you like to discover today?</h1>
                 <form id="question1">
+                    
                     <input type="radio" id="artist" name="answer1" value="artist">
-                    <label for="artist">Artist</label><br>
+                    <label for="artist">Artist</label><br/>
```

Comment:
The line break <br> doesn't need the closing / slash on it

Reply:
okie dokie